### DISCUSSION TITLE
How do I fix this?

### DISCUSSION THREAD

**Discussion by MrMidnightOfficial** [vlang/v#15092] [General]
How do I fix this error:
`./search.v:57:2: error: expression evaluated but not used`'
`   55 | match search_1_user_input {`
`   56 |     '1' {vanilla_ice_cream_breyers_1() or{panic(err)}`
`   57 |     '2' {`
`  58 |         println('soon')`
`   59 |     }`

**Comment by larpon** [comment]
You have a higher chance that someone will answer if you post the full source code that produces the error. This is a rule of thumb for asking for help in general.

Make it as easy as you possibly can for people to be able to reproduce your situation locally.

**Reply by Wajinn** [reply]
It is very hard to understand what the person is doing or trying to do, if they don't show a full example (that produces the error) or their source code.
